src/cascadia/TerminalControl/KeyChord.idl

```diff
@@ -18,6 +19,7 @@ namespace Microsoft.Terminal.Control
         KeyChord();
         KeyChord(KeyModifiers modifiers, Int32 vkey);
         KeyChord(Boolean ctrl, Boolean alt, Boolean shift, Int32 vkey);
+        KeyChord(Boolean ctrl, Boolean alt, Boolean shift, Boolean win, Int32 vkey);

```

Comment:
minimizing diffs so we don't need to update all callers to add a `false`?

Reply:
sure lets call it that 😝